client/views/console-view.js

```diff
@@ -155,6 +148,7 @@ define([
 
         initialize: function() {
             this.$el.addClass("full-height");
+            this.graphics = true;
```

Comment:
graphics should default to false to match the brogue client (if the brogue client defaulted to true, e.g. by some cmd line options) then graphics would be updated by a signal from brogue

Reply:
Noted that this changes doesn't actually stop graphics being displayed. However, the user experience expected is for (play) to not start in tiles mode. To achieve this I also will need to change the cmdline current used to start brogue, which I will do after merging this.